js/src/workspaces/window.js

```diff
@@ -103,7 +103,7 @@
         _this.currentCanvasID = _this.imagesList[0]['@id'];
       }
 
-      this.annoEndpointAvailable = !jQuery.isEmptyObject($.viewer.annotationEndpoint);
+      this.annoEndpointAvailable = !jQuery.isEmptyObject(_this.state.getStateProperty('annotationEndpoint'));
```

Comment:
Here is an example of getting rid of a reference to the Viewer object, and using the global state.